app/controllers/application_controller.rb

```diff
@@ -56,8 +55,11 @@ def after_login_redirect(url_to_redirect = nil)
     redirect_to(url_to_redirect) && return unless url_to_redirect.nil?
     if current_website
       redirect_to errors_url, notice: "Logged in"
+    elsif !current_user.websites.empty?
+      set_website(current_user.websites.first)
+      redirect_to errors_url
```

Comment:
I don't understand why you need this. This should be done in a controller where this is called. You don't set a website here at random. Can you explain?